### PR TITLE
Solve the task

### PR DESCRIPTION
**Pull request by Cerne13**
Driver list page
image

Create driver page (w/licence validation)
image

Driver details page
image

Driver licence update page
image

Driver deletion page
image

Car creation page
image

Car detail page (not mine)
image

Car detail page (mine now. My preciousss...)
image

### CODE REVIEW

**Review by Nikyxa — APPROVED** [review]
Cool!

File: taxi/forms.py [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+
+def licence_validation(license_num):
+    licence_num_len = 8
+    # decided to to all-in-one check so a user will see all the requirements in
+    # case of any validation error
+    if (
+            len(license_num) != licence_num_len
+            or not (license_num[:3].isupper() and license_num[:3].isalpha())
+            or not license_num[4:].isdigit()
+    ):
```
Comment (MasterpieceElbow) [inline taxi/forms.py:17]: It would be better to write 
`if not (here conditions that should be fulfilled joined by 'and') `

File: taxi/urls.py [hunk 2]
```diff
@@ -46,6 +54,18 @@
...
+        "drivers/<int:pk>/update/",
+        DriverLicenceUpdateView.as_view(),
+        name="driver-licence-update"
+    ),
+    path(
+        "drivers/<int:pk>/delete/",
+        DriverDeleteView.as_view(),
+        name="driver-delete"
+    ),
+    path("drivers/<int:pk>/toggle", driver_toggle, name="driver-toggle")
```
Comment (MasterpieceElbow) [inline taxi/urls.py:68]: Always add the following `/` to the path.

**Review by MasterpieceElbow — CHANGES_REQUESTED** [review]

File: taxi/forms.py (hunk 1 above)
Comment (Cerne13) [reply taxi/forms.py:17]: 👍

**Review by TPolina — APPROVED** [review]
Well done!

**Pull request closed without merging**